src/main/java/Bron.java

```diff
@@ -0,0 +1,54 @@
+import java.util.Scanner;
+
+public class Bron {
+    public static void main(String[] args) {
+        String logo = "  ____    ____      ____    _     _\n"
+                    + " |  _ \\  |   _\\   /  __  \\ | \\   | |\n"
+                    + " | |_) | | |_) |  | |  | | |  \\  | | \n"
+                    + " |  _ <  | ___/   | |  | | | |\\\\ | |\n"
+                    + " | |_) | | | \\ \\  | |__| | | | \\\\| |\n"
+                    + " |____/  |_|  \\_\\  \\____/  |_|  \\__|\n";
+        Scanner input = new Scanner(System.in);
+        System.out.println(logo + "Hello! I'm Bron\n" + "What can I do for you?\n");
+
+        Task[] tasks = new Task[100];
+        int taskCount = 0;
+
+        while (true) {
```

Comment:
Should the main function be shortened (eg. with more abstraction)? This `main` function is quite hard to read for me as it is quite long.